package.json

```diff
@@ -33,13 +33,14 @@
     "ramda": "~0.27",
     "vue": "~3",
     "vue-flatpickr-component": "~9",
-    "vue-i18n": "^9.2.2",
+    "vue-i18n": "^10.0.7",
     "vue-router": "~4",
     "xml-formatter": "~2"
   },
   "devDependencies": {
-    "@intlify/vue-i18n-loader": "~4",
+    "@intlify/unplugin-vue-i18n": "^6.0.8",
```

Comment:
#671 has a few small notes about Vue I18n. I'm not quite sure, but I think when I wrote the issue, I was imagining that unplugin-vue-i18n was a replacement for vue-i18n-loader. In actuality, I think it's a replacement for vue-cli-plugin-i18n, which itself uses vue-i18n-loader. I was able to remove all i18n config from the Vue CLI config file, vue.config.js. unplugin-vue-i18n can be used with either Vite (in vite.config.js) or webpack (karma.conf.js).